test/unit/detail/CMakeLists.txt

```diff
@@ -16,8 +16,10 @@ sharg_test (format_help_test.cpp CYCLIC_DEPENDING_INCLUDES include-sharg-detail-
 sharg_test (format_html_test.cpp CYCLIC_DEPENDING_INCLUDES include-sharg-detail-format_help.hpp
                                                            include-sharg-detail-format_man.hpp)
 sharg_test (format_man_test.cpp)
-sharg_test (format_ctd_test.cpp)
-sharg_test (format_cwl_test.cpp)
+if (SHARG_USE_TDL)
```

Comment:
List_unused_unit_tests will warn. Maybe either exclude these from warnings, or have the tests skipped?